test/node.js

```diff
@@ -0,0 +1,46 @@
+import test from 'ava';
+import pEvent from 'p-event';
+import path from 'path';
+import execa from '..';
+
+process.env.PATH = path.join(__dirname, 'fixtures') + path.delimiter + process.env.PATH;
+
+test('node()', async t => {
+	const {exitCode} = await execa.node('test/fixtures/noop');
```

Comment:
You should not need `test/fixtures/` because of the `PATH` fix above?

Reply:
I would agree with you, but apparently with the modified path running `noop` works (instead of `./test/fixtures/noop`) but running `node noop` doesn't work (while `node ./test/fixtures/noop` does work).
I don't know how Node resolves file execution as parameter, there might be another way.

Reply:
Oh yes you're right, `PATH` is only used for the executable file, not its arguments, and here `noop` is actually the argument.